Fix broken reference when an Alembic file is moved (ABC-370) (#548)
Unity-Technologies/com.unity.formats.alembic: com.unity.formats.alembic/CHANGELOG.md

### Fixed
- Prevent a warning in the Console due to deprecated `RecorderInputSetting.ValidityCheck()` when Alembic is installed along with Recorder.
- Prevent warnings related to FindObjectOfType<> calls on package installation with Unity Editor 2023.1+.

> - Prevent broken links in Alembic files that had been imported then moved outside of the Unity Editor.

Reviewer: ```suggestion
- Prevent broken links in imported Alembic files that have been moved outside of the Unity project folder.
```

### Fixed
- Prevent a warning in the Console due to deprecated `RecorderInputSetting.ValidityCheck()` when Alembic is installed along with Recorder.
- Prevent warnings related to FindObjectOfType<> calls on package installation with Unity Editor 2023.1+.

> - Prevent broken links in imported Alembic files that have been moved outside of the Unity project folder.

Reviewer: ```suggestion
- Prevent broken links in Alembic files that would be first imported and then moved outside of the Unity project folder.
```
